azure/iac/sql_alert_policy.rego

```diff
@@ -1,91 +1,141 @@
 package rule
 
 #
-# PR-AZR-0103-ARM
+
 #
 
 # https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/servers/securityalertpolicies
 # https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/managedinstances/securityalertpolicies
 
+# PR-AZR-0102-ARM
+
 default sql_server_alert = null
 
+azure_attribute_absence["sql_server_security_alert_disabled"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.sql/servers"
+    sql_resources := resource.resources[_]
+    lower(sql_resources.type) == "securityalertpolicies"
+    not sql_resources.properties.state
+}
+
+azure_attribute_absence["sql_server_security_alert_disabled"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.sql/servers/securityalertpolicies"
+    not resource.properties.state
+}
+
+
 azure_sql_security_alert_disabled["sql_server_security_alert_disabled"] {
```

Comment:
**This rule is unnecessary i believe. please comment it out so that script ignore it. try to find a real world template reference where it shows securityalertpolicies as a child resource type inside microsoft.sql/servers and if found create a separate rule with that.**